packages/zowe-explorer/__tests__/__integration__/extension.integration.test.ts

```diff
@@ -1120,13 +1102,14 @@ async function getAllNodes(nodes: IZoweTreeNode[]) {
     let allNodes = new Array<IZoweTreeNode>();
 
     for (const node of nodes) {
-        allNodes = allNodes.concat(await getAllNodes(await node.getChildren()));
+        let nodeChildren = await node.getChildren();
+        nodeChildren = nodeChildren.filter((node) => !node.label.includes("No datasets found"));
```

Comment:
should use a different variable here instead of `node` which is causing a shadowed name error